### PR TITLE
feat: /data/mongodb revamp

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated content and layout as per [doc](https://docs.google.com/document/d/1mCztOFZYbC3TeUM7j7pGfSoxKzBs20W1pJHZhdLqKs8/edit) and [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=13-18435&m=dev)

## QA

- View the site locally in your web browser at: https://canonical-com-1348.demos.haus/data/mongodb
- Check against doc and design
  - Don't forget to check smaller screens 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12347

### CODE REVIEW

**Review by britneywwc — COMMENTED** [review]
Left some suggestions and questions. Feel free to ignore the ones that may have been agreed with Design :)

File: templates/data/mongodb/index.html [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+        <p>Canonical's Charmed MongoDB automates operational tasks, so you can spend more time on value-driving activities.</p>
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Automated security and configuration best practice</li>
+          <li class="p-list__item is-ticked">Made for physical, virtual machines (VMs) and Kubernetes</li>
+          <li class="p-list__item is-ticked">Integrated observability for monitoring, alerting and auditing</li>
+          <li class="p-list__item is-ticked">Support for multi-cloud environments</li>
+        </ul>
+        <hr class="p-rule--muted" />
+        <p>
+          <a href="/data/opensearch/managed">Explore Managed MongoDB Solutions&nbsp;&rsaquo;</a>
```
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/data/mongodb/index.html:212]: I'm pretty sure that's an error on the figma as we always add bottom padding to these sections, but @mattea-turic could you confirm?

File: templates/data/mongodb/index.html [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+            <a href="/contact-us" aria-controls="data-mongodb-modal">Contact us <span class="u-off-screen">about self-managed MongoDB solutions</span>&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
```
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/data/mongodb/index.html]: `p-rule` is being deprecated so used the more recent suggestion instead, [pr here](https://github.com/canonical/vanilla-framework/pull/5241/files)

File: templates/data/mongodb/index.html [hunk 3]
```diff
@@ -0,0 +1,509 @@
...
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <p class="p-heading--2">
+        <a href="/blog/author/mtabirao">Updates from the product manager&nbsp;&rsaquo;</a>
```
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/data/mongodb/index.html]: I instinctively updated according to the doc, but now I'm noticing that in other pages we are not capitalizing it 🤔 @Sophie-32 wdyt?

File: templates/data/mongodb/index.html [hunk 4]
```diff
@@ -0,0 +1,509 @@
...
+      <div class="col">
+        <p>
+          MongoDB's architecture and design principles provide a robust and scalable solution for handling diverse and large-scale data requirements.
+        </p>
+        <p>
+          Its document-oriented model offers flexibility, while features like replica sets, sharding, and transactions ensure high availability, scalability, and data integrity. This makes MongoDB a powerful choice for various industries.
+        </p>
+        <hr class="p-rule--muted" />
+        <p>
+          <a href="/data/mongodb/what-is-mongodb">Learn more about MongoDB&nbsp;&rsaquo;</a>
```
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/data/mongodb/index.html:402]: Yup, but looks like it was merged this morning so I've rebased and it should be working now

**Comment by mtruj013** [comment]
Thanks @britneywwc! Addressed almost everything and left couple of qs for the designers for the bits I wasn't sure about, could you take another look?

**Comment by britneywwc** [comment]
Just one more thing, there should be some padding around the card here Screenshot 2024-08-22 at 2 03 05 PM

**Comment by mtruj013** [comment]
@britneywwc it has padding already which is provided by the `p-card--overlay` class 
image

It looks like the figma calls for an additional .5 rem though. I can update, but really it sounds like the card component should be officially upstreamed on Vanilla's side as every version we build of this card will need the additional custom style. Issue [here](https://github.com/canonical/vanilla-framework/issues/5319)

File: templates/data/mongodb/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (mattea-turic) [reply templates/data/mongodb/index.html:213]: @britneywwc @mtruj013 Yes, that's an error on the figma, sorry!

File: templates/data/mongodb/index.html [hunk 5]
```diff
@@ -0,0 +1,509 @@
...
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <section class="p-section--hero">
+    <div class="row--25-75">
+      <div class="col p-image-wrapper u-hide--small">
+        {{ image(url="https://assets.ubuntu.com/v1/b2b3c5d9-MongoDB_Logo.png",
```
[the remark this answers is not in the document]
Comment (mattea-turic) [reply templates/data/mongodb/index.html:22]: @mtruj013 Lmk if this works! https://assets.ubuntu.com/v1/f7fbffe1-mongoDB.png

File: templates/data/mongodb/index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (Sophie-32) [reply templates/data/mongodb/index.html]: Good catch, should be lower case. I changed it in the copydoc.

**Review by britneywwc — APPROVED** [review]
Looks great, thanks! 🔥

**Pull request merged**